tests/component-tests/tableOfContents/js/TableOfContentsTests.js

```diff
@@ -276,7 +276,7 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
             // To address IE7 problem, http://bugs.jquery.com/ticket/7117
             // To fix, strip it URI if the windows.location is in href. Otherwise, do nothing.
             var eleHref = elm.attr("href").replace($(location).attr("href"), "");
-            jqUnit.assertTrue("ToC anchor set correctly", hInfo.url ? eleHref === hInfo.url : eleHref.indexOf(fluid.fluidInstance) === 1);
+            jqUnit.assertTrue("ToC anchor set correctly", hInfo.url ? eleHref === hInfo.url : eleHref.indexOf("fluid-id-" + fluid.fluidInstance) === 1);
```

Comment:
Am I correct in reading this as: if a header had an existing id, verify that the table of contents entry href is using it, otherwise verify that the table of contents entry href has the right form (starts with "fluid-id-" + fluid.fluidInstance) ?

It looks to me like we aren't verifying the generated ids for headers that didn't already have one. Is this right?